src/dir.rs

```diff
@@ -161,6 +186,11 @@ impl Dir {
     /// Currently, we recommend to fallback on any error if this operation
     /// can't be accomplished rather than relying on specific error codes,
     /// because semantics of errors are very ugly.
+    ///
+    /// Note that this method does not resolve symlinks by default, so you may have to call
```

Comment:
This creates unnamed file, so no symlinks to resolve

Reply:
Nice catch! I think I inserted this comment by mistake, as the `#[cfg(not(target_os = "linux"))]` version of this method does not include it.